dotCMS/src/main/java/com/dotmarketing/business/RelationshipAPIImpl.java

```diff
@@ -1,14 +1,200 @@
+    public Relationship byInode(String inode) {
+        return super.byInode(inode);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byParent(ContentTypeIf parent) throws DotHibernateException {
+        return super.byParent(parent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byChild(ContentTypeIf child) throws DotHibernateException {
+        return super.byChild(child);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> dbAll() throws DotHibernateException {
+        return super.dbAll();
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> dbAll(String orderBy, String contentTypeInode) throws DotHibernateException {
+        return super.dbAll(orderBy, contentTypeInode);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public Relationship byTypeValue(String typeValue) {
+        return super.byTypeValue(typeValue);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf type) throws DotDataException {
+        return super.byContentType(type);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf st, boolean hasParent) {
+        return super.byContentType(st, hasParent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Relationship> byContentType(ContentTypeIf type, String orderBy) {
+        return super.byContentType(type, orderBy);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet);
+    }
+
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, boolean hasParent) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet, hasParent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Tree> relatedContentTrees(Relationship relationship, Contentlet contentlet) throws DotDataException {
+        return super.relatedContentTrees(relationship, contentlet);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Tree> relatedContentTrees(Relationship relationship, Contentlet contentlet, boolean hasParent) throws DotDataException {
+        return super.relatedContentTrees(relationship, contentlet, hasParent);
+    }
+
+    @CloseDBIfOpened
+    @Override
+    public List<Contentlet> dbRelatedContent(Relationship relationship, Contentlet contentlet, boolean hasParent, boolean live, String orderBy) throws DotDataException {
+        return super.dbRelatedContent(relationship, contentlet, hasParent, live, orderBy);
+    }
+
+    @Override
+    public boolean isParent(Relationship rel, ContentTypeIf st) {
+        return super.isParent(rel, st);
+    }
+
+    @Override
+    public boolean isChild(Relationship rel, ContentTypeIf st) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1185)